src/main/java/analyzer/exercises/loglevels/UseSubstringMethod.java

```diff
@@ -0,0 +1,37 @@
+package analyzer.exercises.loglevels;
+
+import analyzer.Comment;
+
+import java.util.Map;
+
+/**
+ * @see <a href="https://github.com/exercism/website-copy/blob/main/analyzer-comments/java/log-levels/use_substring_method.md">Markdown Template</a>
+ */
+class UseSubstringMethod extends Comment {
+    private final String callingMethod;
```

Comment:
I'm not sure we should add the calling method to this comment either: if a student doesn't use `substring` in their solution they will get this comment twice, once for each method.

Instead, I think we can just make the comment generic.